### PR TITLE
cloudwatch-logs multibucket support

### PR DESCRIPTION
**Pull request by alvarezallin**
Support multiple buckets and cloudwatch logging

### CODE REVIEW

**Comment by alvarezallin** [comment]
test again please

**Comment by alvarezallin** [comment]
test again

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ provider "aws" {
 }
 
 module "irisanywhere1" {
-  source = "github.com/graymeta/terraform-aws-irisanywhere//asg?ref=v0.0.2"
+  source = "github.com/graymeta/terraform-aws-irisanywhere//asg?ref=v0.0.3"
```
Comment (jasonsattler) [inline README.md:69]: should be v0.0.4

File: asg/cloud_init.ps1 [hunk 2]
```diff
@@ -173,13 +176,22 @@ catch {
     Write-EventLog -LogName IrisAnywhere -source IrisAnywhere -EntryType Error -eventid 1001 -message "Exception establishing autologon configuration"
 }
 
+#CW Config
```
Comment (jasonsattler) [inline asg/cloud_init.ps1:179]: Wondering if we should eliminate all comments in this file unless they are truly needed.  I think there is a size limit on init files we create.  Watcha think?

Another option is to have the init script just setup ENV variables then call a script that is preloaded on our AMI image.

**Review by jasonsattler — APPROVED** [review]
Left a couple of comments.  overall lgtm

File: README.md (hunk 1 above)
Comment (alvarezallin) [reply README.md]: Updated

File: asg/cloud_init.ps1 (hunk 2 above)
Comment (alvarezallin) [reply asg/cloud_init.ps1:179]: Im currently working on the env vars concept now. should be done in a day or so.

**Pull request merged**